newsletter-november-2022.rst

```diff
@@ -0,0 +1,97 @@
+.. post:: Nov 1, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - November 2022
+========================================
+
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-october-2022>`.
+
+
+New features
+------------
+
+The latest updates from our team:
+
+- :doc:`sphinx-rtd-theme 1.1.0 </theme-release-110>` has been released with minor improvements and as a big step towards future releases with larger changes.
+- After rolling out auto-cancelling builds `last month <https://blog.readthedocs.com/cancel-old-builds/>`_, we have already recorded a whopping 10% decrease in builds.
+  We're really happy with how this turned out, not least that it effectively reduces our cloud footprint by 10% 🌱.
+- We started incrementally refactoring our own documentation to match the `Diátaxis framework <https://diataxis.fr/>`_. We will be writing more about this, as we are currently gaining practical experience.
```

Comment:
Would be awesome to write a blog post with all our discussions and decisions made condensed in a page and talking about our experience doing this work 💯

Reply:
Yes, definitely, I think we owe that to ourselves after taking so much time to figure out the best approach :heart:

Reply:
Made a note elsewhere to get back to this :+1: